include/blogPost.php

```diff
@@ -0,0 +1,48 @@
+<?php
+function getArray(){
+    $post = array(
+        array(
+            'postid'=>0,
```

Comment:
This is very nitpicky, but you're using camel casing most places (whereYouCapitalizeTheFirstLetterOfEveryWordExceptTheFirst). `postid` should probably be written as `postId` (my preference) or `postID` to fit with your convention.